src/ts/domHelper/createNode.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '../types';
+
+const createNode = (DOMstring: string, children?: Array<Element | null | false | undefined>) => {
```

Comment:
```suggestion
const createNode = (DOMstring: string, children?: Element[]) => {
```
이미 optional로 표현하고 있어서 이렇게 해도 된답니다!

그리고
```suggestion
const createNode = (DOMstring: string, children?: Element[] = []) => {
```
이렇게 기본값을 정의해주면 어떨까요?

Reply:
그렇네요! 전달하지 않았을때 내부에서 방어코드를 작성하는것보다 훨씬 간결하고 가독성도 좋을것 같네요 감사합니다ㅎㅎ 아직 타입스크립트가 익숙치 않네요ㅠㅠ